iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -609,6 +606,121 @@ public void testAlterChangeColumn() throws IOException {
     Assert.assertArrayEquals(new Object[]{0L, "Brown"}, result.get(0));
     Assert.assertArrayEquals(new Object[]{1L, "Green"}, result.get(1));
     Assert.assertArrayEquals(new Object[]{2L, "Pink"}, result.get(2));
+
+  }
+
+  // Tests CHANGE COLUMN feature similarly like above, but with a more complex schema, aimed to verify vectorized
+  // reads support the feature properly, also combining with other schema changes e.g. ADD COLUMN
+  @Test
+  public void testSchemaEvolutionOnVectorizedReads() throws Exception {
+    // Currently only ORC, but in the future this should run against each fileformat with vectorized read support.
+    Assume.assumeTrue("Vectorized reads only.", isVectorized);
+
+    Schema orderSchema = new Schema(
+        optional(1, "order_id", Types.IntegerType.get()),
```

Comment:
Do we handle complex types?

Reply:
I think we don't support changing those with the Iceberg-Hive integration - @marton-bod  am I right?